doc/sphinx/source/_static/empty

```diff
@@ -0,0 +1 @@
+This directory is empty.
```

Comment:
I think we should keep it in, because we will likely make changes to it among other config files in the future as we make our docs fancier, which *won't* be generated on the fly.